packages/app/src/types/contracts/index.ts

```diff
@@ -2,9 +2,7 @@
 /* tslint:disable */
 /* eslint-disable */
 export type { ExchangeContractAbi } from './ExchangeContractAbi';
-export type { SwayswapContractAbi } from './SwayswapContractAbi';
```

Comment:
I think you can revert this file. Take a look if your forc is on version v0.13

`forc --version` and to update `cargo install forc --force`

Reply:
weird that it keeps changing this file. I'm on forc v0.13
<img width="460" alt="image" src="https://user-images.githubusercontent.com/8636507/169635715-082d475b-9064-4628-889b-01e98ddf320e.png">